libbeat/otelbeat/beatconverter/beatconverter.go

```diff
@@ -108,13 +114,15 @@ func (c converter) Convert(_ context.Context, conf *confmap.Conf) error {
 			return err
 		}
 		out = map[string]any{
-			beatReceiverConfigKey + "::output::otelconsumer": nil,
+			beatReceiverConfigKey + "::output::otelconsumer": map[string]any{
+				"workers": workers,
+			},
```

Comment:
This looks like we still try to read num_workers and use it if present, and default to NumCPUs if it wasn't set.

IMO we always use NumCPUs workers in otelconsumer and don't look at the output configuration at all to obtain that.

We want to take num_workers from the elasticsearch output and configure the elasticsearch exporter to match though, we just don't want it to affect anything inside the pipeline.